src/tasks/robot/TryCatchRobot.java

```diff
@@ -0,0 +1,31 @@
+package tasks.robot;
+
+public class TryCatchRobot {
+
+    public static void moveRobot(RobotConnectionManager robotConnectionManager, int toX, int toY) {
+        RobotConnection connection = null;
+        int counter = 3;
+
+        for (int i = 0; i < counter; i++) {
```

Comment:
you can write:   for (int i = 0; i < 3; i++)     and remove            int counter = 3;